WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -100,6 +101,28 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 				wallet.Synchronizer.UsdExchangeRate, false);
 		}
 
+		ChangeAvoidanceSuggestionViewModel? bnbSuggestion = null;
+
+		bool canBuildWithoutChangeWithBnb = BnBManager.CanCreateChangelessTransaction(wallet.Coins.Confirmed().ToList(), transactionInfo.Amount, out List<SmartCoin>? bnbResult);
```

Comment:
```suggestion
		bool canBuildWithoutChangeWithBnb = BnBManager.CanCreateChangelessTransaction(wallet.Coins.Available().Confirmed().ToList(), transactionInfo.Amount, out List<SmartCoin>? bnbResult);
```
In the codebase I saw that we use `wallet.Coins.Available().Confirmed()` to get back the available coins, probably we should use that too.